index.html

```diff
@@ -989,7 +1005,10 @@ <h3>
           {{PerformanceResourceTiming/encodedBodySize}}, and
           {{PerformanceResourceTiming/decodedBodySize}}. Further, the
           {{PerformanceResourceTiming/nextHopProtocol}} attribute will be set
-          to the empty string.
+          to the empty string. If the <a data-cite=
+          "FETCH#concept-cors-check"> CORS check</a> fails the following
+          attributes will be set to zero:
+          {{PerformanceResourceTiming/responseStatus}}.
```

Comment:
Do we still need this, given the Fetch PR?

Reply:
Don't think so...removed it.